.github/workflows/gcp-x86-java8.yml

```diff
@@ -30,3 +30,15 @@ jobs:
         uses: hashicorp-contrib/setup-packer@v1
         with:
           packer-version: '1.7.8'
+
+      - name: Configure GCP Credentials
+        run: |
+          echo $ENV_FILE > creds.json;
+          cat creds.json
```

Comment:
do you really want to show the file content?

Reply:
justement , je voullais verifier que ca n'apparaissait meme pas dans les log avec un cat

Reply:
moui, c'est vrai que c'est sensé être caché puisque c'est le contenu d'un secret...
c'est dommage de jouer avec le feu, tout de meme